profile.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
This page should have a constraint to be visible by only logged in user. It's missing.